Sources/Views/MessageInputBar.swift

```diff
@@ -74,7 +74,10 @@ open class MessageInputBar: UIView {
             }
             blurView.isHidden = !isTranslucent
             let color: UIColor = backgroundView.backgroundColor ?? .white
-            backgroundView.backgroundColor = isTranslucent ? color.withAlphaComponent(0.75) : .white
+            backgroundView.backgroundColor = isTranslucent ? color.withAlphaComponent(0.75) : color.withAlphaComponent(1.0)
```

Comment:
👍

Reply:
👍

Reply:
Noticed some older inconsistencies, can you actually change this to:
```
let color: UIColor = backgroundView.backgroundColor ?? .inputBarGray
            backgroundColor = .clear
            backgroundView.backgroundColor = isTranslucent ? color.withAlphaComponent(0.75) : color
```
I wanted to make it such that the topStackView can have a different background color which is why we only want to adjust the backgroundView's color

Reply:
@nathantannar4 ,In your opinion , backgroundView's backgroundColor is not equal to MessageInputBar's backgroundColor, we should delete the property observer of backgroundColor

```
    /// Delete the property observer below !!!
    /// Also sets the backgroundView's backgroundColor to the newValue
    open override var backgroundColor: UIColor? {
        didSet {
            backgroundView.backgroundColor = backgroundColor
        }
    }
```

Reply:
@nathantannar4 , as your suggested code, default MessageInputBar's backgroundColor is .inputBarGray, when first set translucent to true, then set to false, MessageInputBar's backgroundColor would always set to .clear, it violate the original setting.

```
let color: UIColor = backgroundView.backgroundColor ?? .inputBarGray
            backgroundColor = .clear
            backgroundView.backgroundColor = isTranslucent ? color.withAlphaComponent(0.75) : color
```